<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор ключей (Fernet)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bGenerateKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать ключ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNameKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла ключа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>155</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходный текст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bEncrypt">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>265</y>
      <width>406</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="laError">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>110</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teText">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>175</y>
      <width>406</width>
      <height>86</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bCopyPathKey">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Скопировать путь к файлу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bOpenFolderKey">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть папку с файлом</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePathDirKey">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bSetPathKey">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать путь</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Папка для сохранения ключей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDefaultPathKey">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить стандартный путь</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSelectKey">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>130</y>
      <width>186</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбранный ключ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSelectKey">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать ключ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDecrypt">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="laErrorText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>176</width>
      <height>76</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teText_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>406</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSelectFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>325</y>
      <width>406</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bSelectFile">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>325</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>305</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл для расшифровки</string>
    </property>
   </widget>
   <widget class="QLabel" name="laError_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>166</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифрованный текст</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
